src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -338,8 +339,36 @@ public static ParsedAlbumInfo ParseAlbumTitleWithSearchCriteria(string title, Ar
 
                 simpleTitle = CleanTorrentSuffixRegex.Replace(simpleTitle, string.Empty);
 
-                var releaseRegex = new Regex(@"\b(?<artist>" + artist.Name + @")\b.*\b(?<album>"+ string.Join("|",album.Select(s=>s.Title).ToList()) + @")\b",
-                    RegexOptions.IgnoreCase | RegexOptions.Compiled);
+                var albums = "";
+                Logger.Debug(album);
+                if (album.Count == 1)
+                {
+                    albums = string.Join(" ", album.Select(s => s.Title).ToList());
+                } else
+                {
+                    albums = string.Join("|", album.Select(s => s.Title).ToList());
+                }
+                var escapedArtist = Regex.Escape(artist.Name);
+                var escapedAlbums = Regex.Escape(albums);
+                StringBuilder albumBuilder = new StringBuilder();
+                foreach(char c in albums)
```

Comment:
What’s this for? Doesn’t the regex.escape(albums) above handle this?

Reply:
No, not on the case of /// \\\. Hence I employed a custom solution. If you are fine with just that case failing, I can remove.